ci/jenkins/test.sh

```diff
@@ -534,10 +556,11 @@ function deliver_antrea {
     export GOROOT=${GOLANG_RELEASE_DIR}/go
     export GOCACHE="${WORKSPACE}/../gocache"
     export PATH=${GOROOT}/bin:$PATH
-
     git show --numstat
     make clean
-    ${CLEAN_STALE_IMAGES}
+    if [[ $TESTBED_TYPE != "kind" && $TESTBED_TYPE != "kind-flexible-ipam" ]]; then
+        ${CLEAN_STALE_IMAGES}
```

Comment:
If I remember correctly, this command was also used to clean up dangling images and previously built Antrea images. Is there a way to ensure they are properly cleaned up in kind testbed? Otherwise, storage will become limited after many jobs have run.